litellm/tests/test_ollama_local.py

```diff
@@ -24,6 +24,14 @@
 
 # asyncio.run(test_ollama_aembeddings())
 
+# def test_ollama_embeddings():
```

Comment:
Can we add a test for this - I believe you can create a mock test for a stubbed ollama endpoint 

want to make sure there are no regressions on this

Reply:
added unit tests to test_ollama.py as well. however, I am not clear why test method calls are commented out.

Reply:
> however, I am not clear why test method calls are commented out.

Those test call ollama locally and are not mock requests

Reply:
tests in `litellm/tests/test_ollama.py` don't hit ollama server. so we can uncomment all of those.